ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -288,6 +288,10 @@ update_secret generic "build-farm-credentials" \
 	"$( format_attachment "build_farm" sa.sinker.build01.config )" \
 	"$( format_attachment "build_farm" sa.kubeconfig )"
 
+# collects all the secrets for ci-operator
+update_secret generic "build-farm-ci-operator-credentials" \
```

Comment:
This secret has nothing to do with the build farm, we should name it to reflect that.

Reply:
Eventually it will be copied to build01 ... or maybe even to all clusters which need `ci-operator`'s config. The prefix `build-farm` reflects this scenario, or no? :)

Reply:
So it is copied to other clusters ... it will the the same name.
Can you give me the name in your mind?

Reply:
This secret contains the `ci-operator` service account token for api.ci, right? Let's name it for that.

Reply:
what about `ci-operator-apici-credentials`?